### PR TITLE
Backup more files

### PR DESCRIPTION
**Pull request by khenderick**

### CODE REVIEW

File: src/update.py [hunk 1]
```diff
@@ -336,7 +337,7 @@ def update_gateway_backend(tarball, date, version):
         if plugins:
             logger.info('Restoring plugins')
             for plugin in plugins:
-                cmd(['mv', '-v', plugin, os.path.join(python_dir, 'plugins')])
+                cmd(['cp', '-r', plugin, os.path.join(python_dir, 'plugins')])
```
Comment (svanscho) [inline src/update.py:340]:
```
man cp 
-v    Cause cp to be verbose, showing files as they are copied.
```

**Review by svanscho — CHANGES_REQUESTED** [review]

File: src/update.py (hunk 1 above)
Comment (svanscho) [reply src/update.py:340]: Perhaps good to add on the other commands too? I assume this will be present in the update logs then.

**Review by svanscho — CHANGES_REQUESTED** [review]

File: src/update.py (hunk 1 above)
Comment (LnL7) [reply src/update.py:340]: I recall something about `-v` not existing on angstrom.

Comment (svanscho) [reply src/update.py:340]: Sigh

Comment (khenderick) [reply src/update.py:340]: @svanscho does this imply the change is not required and the PR can be approved?

**Review by svanscho — APPROVED** [review]

**Pull request merged**